diagnostic.md

```diff
@@ -94,4 +94,6 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+_teamMember['hobby'] = excercise_
+_teamMember['name']` = give: 'Jason', surname: 'Weeks'_
```

Comment:
Close.  teamMember['name'] refers to the object { given: 'Jason', surname: 'Weeks' }.  This is what is known as a "nested object" since it's an object within another object.  Remember that strings need to be wrapped in single or double quotation marks, otherwise they will be interpreted as object names.